services/ml/execution_impl_ie.cc

```diff
@@ -8,16 +8,27 @@
 
 #include "base/strings/string_number_conversions.h"
 #include "services/ml/compilation_delegate_ie.h"
+#include "third_party/libinference_engine/dldt/inference-engine/include/gna/gna_config.hpp"
 #include "third_party/libinference_engine/dldt/inference-engine/include/inference_engine.hpp"
 
 namespace ie = InferenceEngine;
 
 namespace ml {
 
+// TODO(Junwei): GNA device only be opened for one instance of
```

Comment:
Could you please file an issue to track?

Reply:
Tracked in https://github.com/intel/webml-polyfill/issues/1154